puzzlebot_localization:
  ros__parameters:
    localization_update_rate: 50.0
    initial_pose:
      x: -1.2
      y: 1.2
      theta: -1.57